DataManagementSystem/Service/IRODSStorageElementHandler.py

```diff
@@ -579,7 +577,7 @@ def export_getResourceInfo( self , resource = None ):
     """ Send the storage element resource information
 """
 
-    conn , error, userDict = self.__irodsClient( )
+    conn , error, _ = self.__irodsClient()
```

Comment:
In fact _ is the 'canonical' variable name for unused (but mandatory) unpacked variable.